draft-pfeifer-rtgwg-dmpr/draft-pfeifer-rtgwg-dmpr-00.txt

```diff
@@ -187,7 +191,118 @@ Internet-Draft                    DMPR                    September 2017
 
 3.1.  Header
 
-   The binary header description
+3.1.1.  Preamble
+
+   The preamble of a DMPR packet is as follows
+
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |Magic| Reserved|    NextType   |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+                         The preamble of a packet
+
+   Magic
+      A 3-bit Magic: 0b010
+
+   Reserved
+      Reserved for future use
+
+   NextType
+      8-bit unsigned integer.  Defines the type of the immediately
+      following data.  For a NextType of < 128 this data consists an
+      Extension Header.  For a NextType of >= 128 the rest of the packet
+      is payload, the processing is further specified by NextType.
+
+
+
+
+
+
+
+Davies                   Expires March 18, 2018                 [Page 4]
+
+Internet-Draft                    DMPR                    September 2017
+
+
+3.1.2.  Extension Header
```

Comment:
we should define Extension Header and their processing. One sub section of this should define pre-defined extension header like the fragment extension header